frontend/src/app/[locale]/formPrototype/page.tsx

```diff
@@ -0,0 +1,78 @@
+import { Metadata } from "next";
+import TopLevelError from "src/app/[locale]/error/page";
+import NotFound from "src/app/[locale]/not-found";
+import { ApiRequestError, parseErrorStatus } from "src/errors";
+import withFeatureFlag from "src/hoc/withFeatureFlag";
+import { getFormDetails } from "src/services/fetch/fetchers/formFetcher";
+import { formDetail } from "src/types/formResponseTypes";
+import { WithFeatureFlagProps } from "src/types/uiTypes";
+
+import { redirect } from "next/navigation";
+import { GridContainer } from "@trussworks/react-uswds";
+
+import ApplyForm from "src/components/applyForm/ApplyForm";
+import {
+  validateFormSchema,
+  validateUiSchema,
+} from "src/components/applyForm/validate";
+import BetaAlert from "src/components/BetaAlert";
+
+export function generateMetadata() {
+  const meta: Metadata = {
+    title: `Form test for form React JSON Schema`,
+    description: "this is the form",
+  };
+  return meta;
+}
+
+async function FormPage() {
+  let formData = {} as formDetail;
+  const id = "anyform";
```

Comment:
where will this id eventually be coming from? a query param? if so we should probably go ahead and put this page in a dynamic location to allow for that